.github/workflows/release.yml

```diff
@@ -50,4 +50,16 @@ jobs:
         TWINE_USERNAME: __token__
         TWINE_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
       run: twine upload --verbose dist/* || echo 'Version exists'
-      
\ No newline at end of file
+
+    - id: parse_version
+      name: "Parse version"
+      run: |
+        echo "::set-output name=version::$(PYTHONPATH=cognite/transformations_cli poetry run python -m cognite.transformations_cli --version | cut -c 30-)"
```

Comment:
`transformations_cli, version 1.2.11` is printed, we cut `transformations_cli, version ` from that.